base/ops/boot.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash -e
 #This script is run by /sbin/my_init at boot time
+#this replaces the string @@image_url@@ in the webapp config with the value of $IIIF_IMAGE_BASE_URL
+#before nginx starts
+sed -i "s]@@image_internal_url@@]${IIIF_IMAGE_INTERNAL_URL}]g" /etc/nginx/sites-enabled/webapp.conf
```

Comment:
lets do this in the nginx.sh - then we don't need to get it caught up with the dynatrace changes (as the line will be in the nginx.sh boot?

Reply:
Good call